internal/cmd/mongodbflex/instance/describe/describe.go

```diff
@@ -91,6 +92,11 @@ func outputResult(cmd *cobra.Command, outputFormat string, instance *mongodbflex
 		acls := *instance.Acl.Items
 		strings.Join(acls, ",")
 
+		instanceType, err := mongodbflexUtils.GetInstanceType(*instance.Replicas)
+		if err != nil {
+			instanceType = "Unknown"
```

Comment:
maybe worth asking the team what we should show in this case. Not sure if `unknown` is the best option. Could also be just "-" like we show in other cases, or just the empty string

Reply:
This should never happen as only 1, 3 and 9 are allowed for number of replicas, I've added a comment and changed it to be `""`